pkg/loop/internal/relayer/pluginprovider/contractreader/contract_reader_test.go

```diff
@@ -337,7 +337,7 @@ func (it *fakeContractReaderInterfaceTester) GenerateBlocksTillConfidenceLevel(t
 }
 
 func (it *fakeContractReaderInterfaceTester) MaxWaitTimeForEvents() time.Duration {
-	return time.Millisecond * 100
+	return time.Second * 10
```

Comment:
why, this is a 100x increase?

Reply:
Approved, but we should look into optimising these. Which test was this?

Reply:
Any test that uses `MaxWaitTimeForEvents()` was failing in this PR before I made this change